inference/predictor_deployment.py

```diff
@@ -81,16 +86,27 @@ async def __call__(self, http_request: Request) -> Union[StreamingResponse, str]
                 prompts.extend(text)
         else:
             prompts.append(text)
+
         if not streaming_response:
-            return self.predictor.generate(prompts, **config)
+            if self.use_vllm:
+                return await self.predictor.generate_async(prompts, **config)
+            else:
+                return self.predictor.generate(prompts, **config)
+
         if self.use_deepspeed:
             self.predictor.streaming_generate(prompts, self.streamer, **config)
             return StreamingResponse(self.consume_streamer(), status_code=200, media_type="text/plain")
+        elif self.use_vllm:
+            # streaming only support single prompt
+            if isinstance(prompts, list):
+                prompt = prompts[0]
```

Comment:
Would it be better to add some warnings here?

Reply:
It's actually invalid request for streaming multiple prompts. This is a workaround for current situation. I thought the request protocol for simple and other predictors needs to modify to support both single and multiple prompts. so it can be addressed there in #36